.travis.yml

```diff
@@ -4,11 +4,15 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
+
+sudo:
+  false
 
 before_install:
   - phpenv config-rm xdebug.ini
   - composer self-update
-  - composer --prefer-source update
+  - composer --prefer-dist update
```

Comment:
I would also add `'composer validate` to this and all open source FSi bundles, to prevent any schema errors going into Packagist. As far as I know, they run that command there and do not update the package version if errors are found (like `psr-4` and `targetDir` options being in the same `composer.json`).